internal/github/github.go

```diff
@@ -0,0 +1,194 @@
+package github
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/Devleaps/github-secret-synchronizer/internal/vault"
+	"github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/google/go-github/v64/github"
+	"golang.org/x/crypto/nacl/box"
+)
+
+type GitHubWrapper struct {
+	client *github.Client
+
+	appID          int64
+	installationID int64
```

Comment:
```suggestion
	appID          int16
	installationID int16
```

Reply:
Ah the underlying package expects `int64` as type